### ISSUE TITLE
User study feedback: Additional information to include on Known Issue when opening from Build 

### ISSUE THREAD

**Issue by missymessa** [dotnet/arcade#10984]
Customers would like the following information to be included on the Known Issues issue when they are opened from Build Analysis: 

- Pre-populated values from PR
- Links to AzDO logs, Helix logs
- Link to PR that the Known Issue was generated from (this is a feature that Runfo has and it's a nice breadcrumb).

**Comment by ChadNedzlek** [comment]
I don't think we can afford to put all that in the known issue link, URLs are of limited length... We'd really need a service behind it for this to be feasible (rather than just using a GitHub template), that could fetch the information on demand and put it in the issue.  It would also basically double (or more, because it would need to be URL encoded) the size of the build analysis, because all the information would need to get put in the known issue link URL.

**Comment by missymessa** [comment]
Maybe it would be worth creating a new service to make Known Issues more robust for folks.

**Comment by ChadNedzlek** [comment]
Yeah, I think if we really want it to be useful, it probably needs to take them to some form we control that does a few things:
* gathers all that information automatically and includes it in the issue
* encodes the error search information correctly for them
* shows them the number of hits for the issue before they commit it (to make sure it's not accidentally getting every build everywhere, or none)
* ensures whatever string they enter actually matches their PR (which would avoid a lot of false positives)
